### PR TITLE
tutorial component and restyle

### PR DESCRIPTION
**Pull request by 777Prabhjot**
Hey, i have made tutorial section as you told please check it out.

### CODE REVIEW

**Comment by 777Prabhjot** [comment]
@minhngo3818 done please check now

File: client/src/components/resources/resources.css [hunk 1]
```diff
@@ -0,0 +1,206 @@
...
+
+.resource-list-title {
+	position: relative;
+	text-align: left;
+	padding: 0.1rem 0.75rem;
+	padding-left: 0px !important;
+	color: white;
+	border-bottom: 2px solid #ff9617 !important;
+	background-color: transparent !important;
+	font-size: min(4vw, 1.3rem);
```
Comment (minhngo3818) [inline client/src/components/resources/resources.css:125]: can you make it bold for the title font weight?

File: client/src/components/resources/resources.css [hunk 2]
```diff
@@ -0,0 +1,206 @@
...
+	flex-wrap: wrap;
+	margin-top: 10px;
+	z-index: 100;
+}
+
+.tag {
+	margin-right: 10px;
+	margin-top: 5px;
+	margin-bottom: 5px;
+	padding: 0px 5px;
```
Comment (minhngo3818) [inline client/src/components/resources/resources.css:198]: I think the padding should be 2px 8px to make space loosen.

File: client/src/components/resources/resources.css [hunk 3]
```diff
@@ -0,0 +1,206 @@
...
+	font-size: 16px;
+	cursor: pointer;
+	color: white;
+	width: fit-content;
+	text-align: center;
+	border: 1.5px solid #2196f3;
+	border-radius: 2px;
```
Comment (minhngo3818) [inline client/src/components/resources/resources.css:205]: The border weight is thinner than the design but I think yours fit better making the view not too focused on tags. I think the border-radius should be increased to 5px. It looks like a sharp rectangle with 2px.

File: client/src/pages/resources/Resources.jsx [hunk 4]
```diff
@@ -1,6 +1,7 @@
 import { useState, useEffect } from "react";
 import { useLocation } from "react-router-dom";
-import Tutorials from "./Tutorials";
+// import Tutorials from "./Tutorials";
+import Tutorial from "../../components/resources/tutorials";
```
Comment (minhngo3818) [inline client/src/pages/resources/Resources.jsx:4]: keep the plural name

File: client/src/pages/resources/Resources.jsx [hunk 5]
```diff
@@ -59,7 +60,7 @@ function Tab() {
 						tabState === 1 ? "resources-content active" : "resources-content"
 					}
 				>
-					<Tutorials />
+					<Tutorial />
```
Comment (minhngo3818) [inline client/src/pages/resources/Resources.jsx:63]: keep the plural name

File: client/src/components/resources/tutorials.jsx [hunk 6]
```diff
@@ -0,0 +1,63 @@
...
+
+ToggleItem.propTypes = {
+	id: PropTypes.number.isRequired,
+	title: PropTypes.string.isRequired,
+	description: PropTypes.string.isRequired,
+	url: PropTypes.string.isRequired,
+	tags: PropTypes.arrayOf(PropTypes.string).isRequired,
+};
+
+function Tutorial() {
```
Comment (minhngo3818) [inline client/src/components/resources/tutorials.jsx:46]: The component name should be plural because we render multiple tutorial items.

**Review by minhngo3818 — COMMENTED** [review]
There are some trivial stylings that need fixing.`ToggleItem` is a good approach since we can reuse it for reading and organization items. The reading schema is quite different from the other two. See issue #66. If you think it's possible, can you make the `ToggleItem` a separate component and make it more general to apply for tutorial, reading, and organization items? Otherwise, I will make a separate issue or leave all three to have their own component. Please let me know.

**Comment by 777Prabhjot** [comment]
@minhngo3818 done

File: client/src/pages/resources/Resources.jsx [hunk 7]
```diff
@@ -1,6 +1,7 @@
 import { useState, useEffect } from "react";
 import { useLocation } from "react-router-dom";
-import Tutorials from "./Tutorials";
+// import Tutorials from "./Tutorials";
```
Comment (minhngo3818) [inline client/src/pages/resources/Resources.jsx:3]: Please remove this line

**Review by minhngo3818 — CHANGES_REQUESTED** [review]
Make sure to remove the comment.
Excellent!

File: client/src/pages/resources/Resources.jsx (hunk 7 above)
Comment (777Prabhjot) [reply client/src/pages/resources/Resources.jsx]: removed

**Review by minhngo3818 — APPROVED** [review]

**Comment by minhngo3818** [comment]
Solved #128

**Pull request merged**